com.avaloq.tools.ddk.xtext.builder/src/com/avaloq/tools/ddk/xtext/builder/MonitoredClusteringBuilderState.java

```diff
@@ -887,16 +897,6 @@ protected boolean recordDeltaAsNew(final Delta newDelta) {
               newState.register(new DefaultResourceDescriptionDelta(oldDescription, null));
             }
           }
-          // CHECKSTYLE:CHECK-OFF IllegalCatch
-          // If we couldn't load it, there's no use trying again: do not add it to the queue
-        } catch (final Throwable e) {

```

Comment:
i think this catch block has to be brought back... Otherwise, if some exception occurs the whole build will fail